greeter/README.md

```diff
@@ -433,7 +438,7 @@ Again, we are using `bind` and `injection` to bind our app with Interceptor and
 
 ### Step 1: Create a caching service
 
-1.  Assuming this caching service is ready to use, we'll just copy and paste `src/services/caching.service.ts`.
+1.  Assuming this caching service is ready to use, we'll just copy and paste `src/caching.service.ts`.
```

Comment:
> Assuming this caching service is ready to use, we'll just copy and paste `src/caching.service.ts`.

Copy the contents of [caching.service.ts](https://github.com/strongloop/cascon2019/blob/7c25fc4c766404286f6e5a3517f9ab9fe2522e20/greeter/src/caching.service.ts) to a new file in your workspace named `src/caching.service.ts`.